setup.py

```diff
@@ -85,15 +85,15 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
-tests_require=['pytest', 'pytest-cov', 'pytest-timeout >=0.3']
-
+tests_require = ['pytest>=3.6.0', 'pytest-cov', 'pytest-timeout>=1.3.1']
```

Comment:
Is there any reason for enforcing these specific versions for testing? As it's a library, we should require the lowest possible version.